packages/gensx-storage/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@gensx/storage",
+  "version": "0.0.1",
+  "description": "Cloud storage, blobs, sqlite, and vector database providers/hooks for GenSX.",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=18.0.0",
+    "pnpm": ">=9.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gensx-inc/gensx.git",
+    "directory": "packages/gensx-storage"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./blob": {
+      "types": "./dist/blob/index.d.ts",
+      "import": "./dist/blob/index.js"
```

Comment:
I don't see `/blob/index.ts`? What are we exporting from there?

Reply:
i wonder if it hadn't pushed one of my commits when you looked at this file. there's no /blob in there anymore

Reply:
oh yeah that's what it was